scripts/build-rancherd

```diff
@@ -1,6 +1,11 @@
 #!/bin/bash
 set -e
 
+cd $(dirname $0)
+K8S_VERSION=$(grep 'ARG RKE2_VERSION=' ../package/Dockerfile.runtime | sed 's/.*RKE2_VERSION=//')
```

Comment:
I would probably parameterize this further up in the build system, and make it a variable that is passed as a build arg to `docker build -t ${RUNTIME_IMAGE} -f Dockerfile.runtime` in `/scripts/runtime`, and in this script... but if you're looking for a quick fix this probably works fine.

Either way, I would probably also call it RKE2_VERSION instead of K8S_VERSION since this will have a +rke2r1 suffix on it.

Reply:
I will update this as you suggest.